### ISSUE TITLE
IO not mocked (specifically, IO.read not mocked)

### ISSUE THREAD

**Issue by smtlaissezfaire** [fakefs/fakefs#461]
See:

https://gist.github.com/smtlaissezfaire/d0f71b90bea3bee6fa5e3a0c3f700954

**Comment by grosser** [comment]
yeah it's not ... not sure if that's a bug or feature ... you can try making a PR and see if it works out nicely
